src/tslint.ts

```diff
@@ -86,6 +86,21 @@ class Linter {
     private containsRule(rules: Lint.RuleFailure[], rule: Lint.RuleFailure) {
         return rules.some((r) => r.equals(rule));
     }
+
+    private getRulesDirectories(): Array<string> {
```

Comment:
Done in commit https://github.com/palantir/tslint/commit/ee60a0cd1b858bd9555f21e9b4efdb501ebb570d.